arkouda_distance_wserver/server/ServerModules.cfg

```diff
@@ -1 +1 @@
-DistanceCalcMsg
\ No newline at end of file
+distanceMsg
```

Comment:
Does the update require the module name or the file name in `ServerModules.cfg`. This should probably be updated so that the 2 match though.

Reply:
It is required that the name in `ServerModules.cfg` match the Chapel module for it to build. This is not new with this change. The `arkouda_distance_wserver` code I don't believe ever would have compiled previously since the module name did not match the name give in `ServerModules.cfg`.

Reply:
Oh, I see what you mean. Chapel files should always match the name of the module, I did not realize that they were different in this case (I actually thought it was a requirement that they match).